cloud/aws/codes.yaml

```diff
@@ -10,9 +10,5 @@
 "Fundamentals": "default"
 "Practitioner": "practitioner"
 "Puppet Essentials for Windows": "windows"
-"Virtual Puppetizing Infrastructure 1": "puppetize-virtual-classroom-1"
-"Virtual Puppetizing Infrastructure 2": "puppetize-virtual-classroom-2"
-"Virtual Puppetizing Infrastructure 3": "puppetize-virtual-classroom-3"
-"Puppetizing Infrastructure 1": "puppetize-virtual-classroom-1"
-"Puppetizing Infrastructure 2": "puppetize-virtual-classroom-2"
-"Puppetizing Infrastructure 3": "puppetize-virtual-classroom-3"
+"Virtual Puppetizing Infrastructure": "puppetize-virtual-classroom"
+"Puppetizing Infrastructure": "puppetize-virtual-classroom"
```

Comment:
Looked like the PI course can be named differently in certain cases, so this makes sure that both name variants are able to boot the classroom with the correct machines.